Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/internal/converters/ContactDiaryRoomConverters.kt

```diff
@@ -25,4 +26,18 @@ class ContactDiaryRoomConverters {
     fun fromJodaDuration(duration: Duration?): Long? {
         return duration?.millis
     }
+
+    @TypeConverter
+    fun toTestType(value: String?): ContactDiaryTestEntity.TestType? =
+        value?.let { ContactDiaryTestEntity.TestType.values().single { it.raw == value } }
```

Comment:
`singleOrNull` ? single throws an exception if not found

Reply:
updated